source/Makefile

```diff
@@ -8,7 +8,7 @@ clean:
 CPPFLAGS += -I$(TARGET_OSSL_INCLUDE_PATH) -I.
 # For second include path, i.e. out of tree build of OpenSSL uncomment this:
 # CPPFLAGS += -I$(TARGET_OSSL_INCLUDE_PATH2)
-CFLAGS += -pthread
+CFLAGS += -pthread -O2
```

Comment:
Why not `O3`?

Reply:
Thought the -O3 too aggressive which may hurt the performance. It has been many years I've read about the